### PR TITLE
New search UI improvements

### PR DESCRIPTION
**Pull request by ishouty**
* Fix issue with search tabs href not scrolling to the top but keep scroll state for all platforms
* Add padding around sort and pagination area for search for mobile
* Add feature for mobile pagination total results not allow to show pagination
* Hide total results of pagination of the bottom for mobile images and library
* always make the pagination align to the right side for all platforms
* vertically align pages numbers to pagination icon for all platforms
Screenshot 2020-12-02 at 17 37 19
Screenshot 2020-12-02 at 17 55 16
Screenshot 2020-12-02 at 17 55 40

### CODE REVIEW

File: catalogue/webapp/pages/works.tsx [hunk 1]
```diff
@@ -318,6 +318,7 @@ const Works: NextPage<Props> = ({
                                 window.scrollTo(0, 0)
                               );
                             }}
+                            hideMobileTotalResults={true}
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/works.tsx:321]: Maybe I have misread this - but are we hiding total results on mobile in all cases?

Comment (ishouty) [reply catalogue/webapp/pages/works.tsx:321]: ahh nah we have two paginations which contain totalResults. We will hide the bottom total results for the mobile

Comment (jamesgorrie) [reply catalogue/webapp/pages/works.tsx:321]: But we hide it on works and images?

File: common/views/components/Paginator/Paginator.js [hunk 2]
```diff
@@ -89,20 +98,32 @@ const Paginator = ({
     <Fragment>
       <Space
         h={{ size: 'm', properties: ['margin-right'] }}
```
Comment (jamesgorrie) [inline common/views/components/Paginator/Paginator.js:100]: Could we just add the class to the space component? Why do we need a new wrapper?

**Review by jamesgorrie — APPROVED** [review]
All good, just nitpicking.

File: catalogue/webapp/pages/works.tsx (hunk 1 above)
Comment (ishouty) [reply catalogue/webapp/pages/works.tsx:321]: yes we will hide this on works and images for mobile.

Comment (ishouty) [reply catalogue/webapp/pages/works.tsx]: Ive added searchPrototype toggle to this to make sure it only displays only in the new search.

**Pull request merged**